packages/vscode-apollo/create-server-symlink.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+const symlinkDir = require("symlink-dir");
+const path = require("path");
+
+symlinkDir("../../apollo-language-server", "./node_modules/")
+  .then(result => {
+    console.log(result);
+  })
+  .catch(err => {
```

Comment:
We should be able to remove this catch if we're only rethrowing the error and not doing anything with it